card/lib/card/query/value.rb

```diff
@@ -14,18 +15,27 @@ def initialize rawvalue, query
       def parse_value rawvalue
         case rawvalue
         when String, Integer then ["=", rawvalue]
-        when Array           then [rawvalue[0], rawvalue[1..-1]]
+        when Array           then parse_array_value rawvalue
         else raise("Invalid Condition Clause #{rawvalue}.inspect}")
         end
       end
 
+      def parse_array_value array
+        operator = operator?(array.first) ? array.shift : :in
+        [operator, array]
```

Comment:
this is where the change is.

Reply:
Doesn't matter anymore but what's the difference between `[rawvalue[0], rawvalue[1..-1]]` and `rawvalue`?

Reply:
it's the difference between `[op, v1, v2]` and `[op, [v1, v2]]`.  definitely easier to see that now! :)